<?xml version="1.0"?>
<launch>
    <param name="use_sim_time" value="true"/>

    <!--Nodes-->
	<node name="path_planner_plotting" pkg="fs_planning-2022" type="plotting.py" output="screen"/>

    <!--Topics-->
    <param name="planner/map_topic" value="/planner/cones"/> 
    <param name="planner/pose_topic" value="/planner/pose"/>
    <param name="planner/waypoints_topic" value="/planner/path"/> 

    <!--Cones Styles-->
    <param name="planner/blue_cone_style" value="0"/>
    <param name="planner/yellow_cone_style" value="1"/>
    <param name="planner/orange_cone_style" value="2"/>
    <param name="planner/big_cone_style" value="3"/>
    <param name="planner/unkown_cone_style" value="4"/>

</launch>

